test/unit/endpoint.test.js

```diff
@@ -0,0 +1,27 @@
+/* global describe it */
```

Comment:
Please rename this file to `generate.test.js` - tets files should have names which are similar to endpoint file name.